bin/wmagent-resource-control

```diff
@@ -167,13 +167,12 @@ def printThresholds(resourceControl, desiredSiteName):
         print "%s - %d running, %d pending, %d running slots total, %d pending slots total%s:" % (
                         siteName, runningJobs, pendingJobs, runningSlots, pendingSlots, stateMsg)
 
-        for task in siteThresholds['thresholds']:
+        for task, thres in siteThresholds['thresholds'].iteritems():
```

Comment:
I don't think so. In python 3 items() is the same as what iteritems() is in python 2. In python 2 both exist and items() actually builds the list. Use iteritems() for now, the python 2to3 converter will make it items later. (It will translate ```items()``` to ```[items()]```)